pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -65,7 +65,9 @@ type GCECloud struct {
 	service           *compute.Service
 	containerService  *container.Service
 	projectID         string
-	zone              string
+	region            string
+	instanceZone      string
+	managedZones      []string
```

Comment:
That would be ideal, so that switching a cluster between single zone and multi-zone (or vice versa) wouldn't require restarting the controller manager.